services/order.py

```diff
@@ -0,0 +1,31 @@
+from datetime import datetime
+
+from django.db import transaction
+
+from db.models import Ticket, Order, User
+
+
+def create_order(
+        tickets: list[dict],
+        username: str,
+        date: datetime = None
+) -> None:
+    with transaction.atomic():
+        user = User.objects.get(username=username)
```

Comment:
there was an issue in the Check list to use get_user_model() instead of User

Reply:
Fixed